### ISSUE TITLE
Make example more interesting

### ISSUE THREAD

**Issue by RobH123** [unfoldingWord/string-punctuation-tokenizer#67]
The example 'Hello world, my name is Manny!' in the ReadMe leaves me with a few questions.

Could it possibly be extended to 'Hello world, my name is “bŏt#5”, and I don’t know Jess’ phone-number!'

Just curious. Thanks.

**Comment by birchamp** [comment]
what questions does it leave you with specifically @RobH123 ? Are you saying you want a more complicated example?

**Comment by birchamp** [comment]
## With punctuation:
![image.png](https://images.zenhubusercontent.com/5cf53901e39e5f33e3e129fa/40d62c76-ed76-413a-b70b-0c8680d9cf26)

## Without punctuation:
![image.png](https://images.zenhubusercontent.com/5cf53901e39e5f33e3e129fa/4a9f00c1-20b3-4ab6-9a7e-cb1b575aef46)

**Comment by RobH123** [comment]
Yes @birchamp, I thought that the given example didn't display enough about how it works (especially when the playground link didn't work). I would recommend the above as a second example.

**Comment by RobH123** [comment]
Oh, I see, the playground shows that it has many optional parameters! The ReadMe gave zero indication of that. I think that would be more important to mention in the ReadMe. Then my particular test string becomes less important.
